.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
We should also include a `feature_report.md` if possible: https://github.com/matter-labs/era-test-node/blob/main/.github/ISSUE_TEMPLATE/feature_report.md